Builds failing because of verify process error

Builds for https://github.com/NNPDF/nnpdf/pull/645 and https://github.com/NNPDF/nnpdf/pull/647 seem to be failing because of the following error:

```
filename = '/Users/travis/miniconda3/conda-bld/nnpdf_1581534117931/_test_env_placehold_placehold_placehold_placehold_placehold_pl...hold_placehold_placehold_placehold_placehol/share/NNPDF/data/commondata/DATA_ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_PT.dat'
sysfile = '/Users/travis/miniconda3/conda-bld/nnpdf_1581534117931/_test_env_placehold_placehold_placehold_placehold_placehold_pl...hold_placehold_placehol/share/NNPDF/data/commondata/systypes/SYSTYPE_ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_PT_DEFAULT.dat'
    def ReadFile(filename, sysfile):
        """
        ReadFile(std::string const & filename, std::string const & sysfile) -> CommonData
    
        Parameters
        ----------
        filename: std::string const &
        sysfile: std::string const &
    
        """
>       return _nnpdf.CommonData_ReadFile(filename, sysfile)
E       RuntimeError: CommonData::VerifyProc: process  is unsupported.
```

This problem was found in https://github.com/NNPDF/nnpdf/pull/632 but @enocera addressed it such that the build then passed. At first glance it's strange because the process defined for ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_PT is HQP_PTQ, which is defined in the list of processes in libnnpdf/src/commondata.cc.

--- a/validphys2/src/validphys/tests/test_loader.py
+++ b/validphys2/src/validphys/tests/test_loader.py
@@ -5,7 +5,7 @@ Test loading utilities.
 """
 import numpy as np
 from hypothesis.strategies import sampled_from, sets, composite
-from hypothesis import given
+from hypothesis import given, settings
 
 from validphys.core import Cuts, CommonDataSpec
 from validphys.loader import Loader, rebuild_commondata_without_cuts
@@ -27,6 +27,7 @@ def commodata_and_cuts(draw):
 ipath = 0
 
 @given(arg=commodata_and_cuts())
+@settings(deadline=None)
 def test_rebuild_commondata_without_cuts(tmp, arg):
     #We have to do this because otherwise files get mixed together. Note that
     #tmp does fire once for all hypothesis runs.

